sample-compose/app/src/main/java/co/nimblehq/sample/compose/extensions/ContextExt.kt

```diff
@@ -0,0 +1,7 @@
+package co.nimblehq.sample.compose.extensions
+
+import android.content.Context
+import android.widget.Toast
+
+fun Context.showToast(message: String) =
+    Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
```

Comment:
(Optional) I think we should make all the variables flexible for different usage 😄 
```suggestion
fun Context.showToast(message: String, length: Int = Toast.LENGTH_SHORT) =
    Toast.makeText(this, message, length).show()
```

Reply:
Actually, we can make it more flexible when used in an actual project. Let me help to make it available from the template.

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/449/commits/068afe2c3dbb379f7947df52637aa9f24505c1ac